opensrp-chw/src/ba/java/org/smartregister/chw/activity/AncMemberProfileActivityFlv.java

```diff
@@ -2,7 +2,7 @@
 
 public class AncMemberProfileActivityFlv extends DefaultAncMemberProfileActivityFlv {
     @Override
-    public Boolean flvSetFamilyLocation() {
+    public Boolean hasFamilyLocationRow() {
```

Comment:
is it possible to check if the memberObject has gps at this point?

Reply:
How?